server/app/routes/users/index.js

```diff
@@ -97,7 +116,7 @@ router.post('/:userId/movie', (req, res, next) => {
     } else {
         allowance = 3;
     }
-    req.newUser.movieQueue.addToQueue(req.body.movieId, allowance)
+    req.newUser.movieQueue.addToQueue(req.body.movieId, allowance, req.newUser)
```

Comment:
Thanks for setting up email Henry. I'm going to wait to merge until we go over it tomorrow so you can walk me through it